### PR TITLE
Add possibility to filter out files after they have been picked for unrecognized mime types

### PR DESCRIPTION
**Pull request by AlexandreSi**
To fix https://forum.gdevelop.io/t/unable-to-upload-3d-models/48566 and https://forum.gdevelop.io/t/bug-can-t-upload-3d-models/48483

### CODE REVIEW

File: newIDE/app/src/ResourcesList/FileToCloudProjectResourceUploader.js [hunk 1]
```diff
@@ -39,19 +39,40 @@ const resourceKindToInputAcceptedMimes = {
...
+    // after the files have been picked.
+    // 'model/gltf-binary'
+  ],
 };
 
-export const getInputAcceptedMimesAndExtensions = (
-  resourceKind: ResourceKind
-) => {
+const getFileNameExtension = (fileName: string): string => {
+  return fileName.split('.').reverse()[0];
```
Comment (4ian) [inline newIDE/app/src/ResourcesList/FileToCloudProjectResourceUploader.js:51]: Consider using `path.extname` from `path-browserify` (NOT from `optionalRequire('path')`). Note that `path.extname("something.glb")` will return `.glb`

**Review by 4ian — APPROVED** [review]

File: newIDE/app/src/ResourcesList/FileToCloudProjectResourceUploader.js [hunk 2]
```diff
@@ -203,7 +246,12 @@ export const FileToCloudProjectResourceUploader = ({
               onChange={event => {
                 const files = [];
                 for (let i = 0; i < event.currentTarget.files.length; i++) {
-                  files.push(event.currentTarget.files[i]);
+                  if (
+                    !shouldValidateFilePostPicking ||
+                    validateFilePostPicking(event.currentTarget.files[i])
+                  ) {
+                    files.push(event.currentTarget.files[i]);
+                  }
```
Comment (D8H) [inline newIDE/app/src/ResourcesList/FileToCloudProjectResourceUploader.js:254]: Do users know why some files are excluded?

Comment (AlexandreSi) [reply newIDE/app/src/ResourcesList/FileToCloudProjectResourceUploader.js:254]: You mean "Will they know why a file has been excluded from their selection if the extension does not match?" In that case, no. I'll see if I can make it clear.

Comment (AlexandreSi) [reply newIDE/app/src/ResourcesList/FileToCloudProjectResourceUploader.js]: I added a message to inform the user: 
image

**Pull request merged**